lib/suspenders/app_builder.rb

```diff
@@ -227,7 +227,7 @@ def replace_gemfile
     end
 
     def set_ruby_to_version_being_used
-      create_file '.ruby-version', "#{Suspenders::RUBY_VERSION}\n"
+      create_file '.ruby-version', "#{Suspenders::RUBY_VERSION}@#{app_name}\n"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.